.gitlab/build_tioga.yml

```diff
@@ -35,19 +34,34 @@
 
 ####
 # PR Build jobs
-tioga-clang_14_0_0_hip_5_1_1-src:
+tioga-clang_16_0_0_hip_5_6_0-src:
   variables:
-    COMPILER: "clang@14.0.0_hip"
+    COMPILER: "clang@16.0.0_hip"
+    HOST_CONFIG: "tioga-toss_4_x86_64_ib_cray-${COMPILER}.cmake"
+  extends: .src_build_on_tioga
+  allow_failure: true
+
+tioga-cce_15_0_1_hip_5_4_3-src:
+  variables:
+    COMPILER: "cce@15.0.1_hip"
     HOST_CONFIG: "tioga-toss_4_x86_64_ib_cray-${COMPILER}.cmake"
   extends: .src_build_on_tioga
   allow_failure: true
 
 ####
 # Full Build jobs
-tioga-clang_14_0_0_hip_5_1_1-full:
+tioga-clang_16_0_0_hip_5_6_0-full:
```

Comment:
Do we currently have any CI jobs that run the `-full` build jobs?

Reply:
> Do we currently have any CI jobs that run the -full build jobs?

We do have nightly full build jobs running on develop, but they aren't currently in a passing state:
https://lc.llnl.gov/gitlab/axom/axom/-/pipelines/305859

Reply:
Thanks @bmhan12 -- looks like we need to apply the sbang trick

> ==> Error: SbangPathError: Install tree root is too long. Spack cannot patch shebang lines when script path length (129) exceeds limit (127).

Reply:
yea we need to fix these... its on my todo list but low priority